Cabal/changelog

```diff
@@ -1,7 +1,7 @@
 -*-change-log-*-
 
 1.25.x.x (current development version)
-	* Dropped support for versions of GHC earlier than 6.12 (#3111).
+	* Dropped support for versions of GHC earlier than 7.6 (#3833).
```

Comment:
#3111 was a bit different, I'd add a separate line saying "Cabal/cabal-install can no longer be compiled with GHC < 7.6" instead. Otherwise it should be possible to use 2.0 with 7.4.